### PR TITLE
Transfer card names

### PR DESCRIPTION
**Pull request by enjeyw**
**Describe the Pull Request**

Adds name to transfer cards and improves query efficiency.

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Review by tristanhcole — COMMENTED** [review]
Tested on dev. Displays caregiver name correctly. See comments.

File: app/server/schemas.py [hunk 1]
```diff
@@ -332,11 +334,34 @@ class TransferCardSchema(SchemaBase):
...
+                'last_name': obj.user.last_name
+            }
+
+    def get_caregiver(self, obj):
+        for ca in obj.user.custom_attributes:
+            if ca.name == 'caregiver':
+                val = json.loads(ca.value)
+                if val is not True:
+                    return val
+        return None
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply app/server/schemas.py:360]: I'm not sure what the timeline on updating the mobile apps of the smaller vendors is....

File: app/migrations/versions/7edae859c19f_.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    op.add_column('transfer_card', sa.Column('token_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'transfer_card', 'token', ['token_id'], ['id'])
+
+    op.execute("""
+    UPDATE transfer_card
+    SET token_id = transfer_account.token_id
+    FROM transfer_account
+    WHERE transfer_card.transfer_account_id = transfer_account.id;
+    """)
+    # ### end Alembic commands ###
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply app/migrations/versions/7edae859c19f_.py:32]: hmmmmmm it shouldn't be. are u still getting the error?

[the remark this answers is not in the document]
Comment (tristanhcole) [reply app/migrations/versions/7edae859c19f_.py:32]: Yes, still getting that error on dev

File: app/server/schemas.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tristanhcole) [reply app/server/schemas.py:360]: ergh non-preferable but ok

**Pull request closed without merging**